tests/test_import_mlrun.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2025 Iguazio
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import subprocess
+
+acceptable_stderr_errors = [
+    "Kubeflow Pipelines (KFP) is not installed. Using noop implementations."
+]
+
+
+def test_import_mlrun():
+    out = subprocess.run(["python", "-c", "import mlrun"], capture_output=True)
```

Comment:
I get what you tested here, but will it catch regressions in the CI?
It will work when evidently is installed, but it's not the case unless one explicitly installs it

Reply:
Right, for this to work as a regression test for [ML-8951](https://iguazio.atlassian.net/browse/ML-8951) (which is not its only purpose), evidently needs to be installed when the tests run. Added it to `dev-requirements.txt`.